addons/mail/static/src/discuss/core/web/thread_actions.js

```diff
@@ -15,13 +15,12 @@ threadActionsRegistry.add("expand-discuss", {
     name: _t("Open in Discuss"),
     open(component) {
         component.threadService.setDiscussThread(component.thread);
-        component.actionService.doAction(
-            {
-                type: "ir.actions.client",
-                tag: "mail.action_discuss",
-                name: _t("Discuss"),
-            },
-            { clearBreadcrumbs: true }
+        component.actionService.doAction({
+            type: "ir.actions.client",
+            tag: "mail.action_discuss",
+            name: _t("Discuss"),
+        },
+        { clearBreadcrumbs: Boolean(component.homeMenuService?.hasHomeMenu) }
```

Comment:
Why/when do we want this clear exactly?

Reply:
We clear breadcrumbs if we come from enterprise home menu using "open in discuss" button of a chatwindow
<img width="1788" alt="Screenshot 2024-03-28 at 15 54 41" src="https://github.com/odoo/odoo/assets/80891222/3ecf9559-2730-4d7a-b5b2-5a3d12fdd18c">

Reply:
For proper modular code this condition should be moved to `mail_enterprise` then.

What happens exactly if we don't clear it that way?
Isn't it better to let the default web client flow work?

Reply:
Asking PAJU